crates/bevy_gltf/src/loader.rs

```diff
@@ -280,6 +280,7 @@ fn load_material(material: &Material, load_context: &mut LoadContext) -> Handle<
             albedo: Color::rgba(color[0], color[1], color[2], color[3]),
             albedo_texture: texture_handle,
```

Comment:
I propose to rename `albedo` to `base_color_factor` and `albedo_texture` to `base_color_texture`, because it actually doubles as the specular for metallic materials and the diffuse for non-metallic materials.